### ISSUE TITLE
never add markers while sneaking

### ISSUE THREAD

**Issue by RealMuffinTime** [MiraculixxT/bluemap-banner#3]
Currently you can place a banner while sneaking, when the player is at a marker limit, without adding a marker to the map.

I think it would be nice, that when you sneak, there will never be a marker created.

**Comment by MiraculixxT** [comment]
Hey, while adding your idea, i was thinking about sneaking while placing. Isn't sneaking to place a banner a very common thing, because some places dont allow placing it without?

Do you think it would annoy more then help and #2 is maybe a better approach?

**Comment by RealMuffinTime** [comment]
Yeah, I also thought about that, but I think mostly you place banners onto blocks which are not interactable?

Maybe a setting regarding that behavior would be a solution.
Or always when sneaking there is a message, which tells the player no marker has been added, because the player sneaked.
(Maybe, I don't know the effort it would need, add a clickable command to the message to allow sneaking and placing temporarily.)

Basically it doesn't need to be sneaking to achieve no adding, but it would be nice to have some easy way to not add markers, while placing banners.

But the blacklist would be server wide, and not customisable for an individual player, or?

**Comment by MiraculixxT** [comment]
Sooo, i added it finaly :) (a bit late, sorry for that)
I decide to make it configurable inside the config, whether sneaking should be ignored or not. Together with the custom blocked names list i hope it works for you!

Update is downloadable on Modrinth now, you might need to delete your settings.yml.

**Comment by RealMuffinTime** [comment]
Perfect.
